spec/validators/expiry_date_validator_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe ExpiryDateValidator, type: :validator do
+  let(:validator) { ExpiryDateValidator.new }
+  describe('#validate') do
+    subject(:date_validation) { validator.validate(bet) }
+    let(:user) do
+      User.create(username: 'Ivan',
+                  email: 'imustbreakyou@mail.com',
+                  password: 'hammerandsickle')
+    end
+    let(:bet) do
+      Bet.new(bet: 'Test bet',
+              creator_id: user.id,
+              expires_at: expiry_date)
+    end
+    context 'valid expiry date' do
+      let(:expiry_date) { DateTime.tomorrow }
+      it 'does not add errors to the record' do
+        expect { date_validation }.to_not change { bet.errors.size }
```

Comment:
You can use the `be_valid` matcher instead to ensure all the ActiveRecord validations are satisfied.